Functions/src/main/java/com/epam/engx/cleancode/functions/task4/Order.java

```diff
@@ -11,18 +11,21 @@
 
     public Double getPriceOfAvailableProducts() {
         double orderPrice = 0.0;
+        removeAvailableProduct();
```

Comment:
That the reason why it is bad practice to use `!` in conditions. Because the purpose of method are opposite - remove NOT available products